debian/postinst

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
```

Comment:
I believe this would be re-run on every upgrade if the recovery-environment package's version is bumped. I think what we wanted is to have this run exclusively during the initial image creation, right?

Reply:
We want to re-copy over major files on upgrade, so that the recovery environment doesn't go too far out of date with the rest of the system.